WindowsDevicePortalWrapper/WindowsDevicePortalWrapper.Shared/HoloLens/HolographicPerception.cs

```diff
@@ -156,6 +153,17 @@ public enum SimulationControlStreamPriority
             await this.PostAsync(HolographicSimulationModeApi, payload);
         }
 
+        /// <summary>
+        /// Compares the current simulation control mode with the expected mode.
+        /// </summary>
+        /// <param name="expectedMode">The simulation control mode that we expect the device to be in.</param>
+        /// <returns>The simulation control mode.</returns>
+        private async Task<bool> VerifySimulationControlMode(SimulationControlMode expectedMode)
```

Comment:
Should this be VerifySimulatrionControlModeAsync? I see it's private so maybe it doesn't matter, but probably still good to keep to the pattern.